mono/utils/mono-compiler.h

```diff
@@ -194,5 +194,12 @@ ssize_t sendfile (int out_fd, int in_fd, __mono_off32_t* offset, size_t count);
 #endif /* __ANDROID_API__ < 21 */
 #endif /* HOST_ANDROID && ANDROID_UNIFIED_HEADERS */
 
-#endif /* __UTILS_MONO_COMPILER_H__*/
+#if defined (__clang__)
```

Comment:
This ended up not used in the end, could be removed if preferred.

Reply:
please remove it

Reply:
It is the current way now -- it had a crucial typo earlier. Worked in CI. Not clear working locally.

Reply:
It had a typo and the solution presently, seen to work in CI but not locally (arguably CI more important).